arviz/plots/backends/matplotlib/distplot.py

```diff
@@ -86,20 +93,50 @@ def plot_dist(
     return ax
 
 
-def _histplot_mpl_op(values, values2, rotated, ax, hist_kwargs):
+def _histplot_mpl_op(values, values2, rotated, ax, hist_kwargs, is_circular):
     """Add a histogram for the data to the axes."""
+
+    bins = hist_kwargs.pop("bins")
+
+    if is_circular:
+
+        if values.min() < np.pi and values.max() > np.pi:
```

Comment:
It is safer to let the user state if the values are degrees or radians, than trying to guess it.